docroot/sites/now.uiowa.edu/modules/now_migrate/src/Plugin/migrate/source/Achievement.php

```diff
@@ -203,11 +215,16 @@ private function calloutReplace($match) {
     // things off in the new callout component.
     $match[2] = preg_replace('|(<br>)+|is', '<br>', $match[2]);
 
+    // Remove anything after the first '<br>',
+    // as it is not in the same '<strong>' group
+    // as the ones on the first line.
+    $headline_match_string = preg_replace('%(<br>|<br \/>|<br\/>).*%is', '', $match[2]);
+
     // Look for a headline to use in the callout, which are bolded strings
     // at the start of the callout. Also look for any additional
     // line breaks. Like before, here they are unnecessary.
     $headline = '';
-    if (preg_match('|^<strong>(.*?)<\/strong>(<br>)*|is', $match[2], $headline_matches)) {
+    if (preg_match_all("|<strong>(.*?)<\/strong>(<br>)*|is", $headline_match_string, $headline_matches)) {
```

Comment:
Taking a look at this. It works great in most cases, but there's not always a `<br>` there, like in https://now.uiowa.edu/2023/02/ui-named-top-producer-fulbright-us-students-seventh-year. We can fix that case by adding `<ul>` to the regex matching options, but not sure if there might be other instances like this with other things.

Reply:
> We can fix that case by adding <ul> to the regex matching options...

Did this. Checking through to see if there are other cases.